alpha/scripts/utils/updateKuserRegisterInfoGroupId.php

```diff
@@ -0,0 +1,40 @@
+<?php
+require_once('/opt/kaltura/app/alpha/scripts/bootstrap.php');
+
+if ($argc < 3)
+{
+	die("Usage: php $argv[0] groupId kuserIdsFile <realrun | dryrun> \n");
+}
+
+$groupId = $argv[1];
+$kuserIdsFile = $argv[2];
+$dryrun = true;
+if($argc == 4 && $argv[3] == 'realrun')
+{
+	$dryrun = false;
+}
+
+$kuserIds = file ($kuserIdsFile) or die ('Could not read file'."\n");
+
+foreach ($kuserIds as $kuserId)
+{
+	$kuserId = trim($kuserId);
+	$kuser = kuserPeer::retrieveByPK($kuserId);
+	if ($kuser)
+	{
+		$registerInfo = $kuser->getRegistrationInfo();
+		if ($registerInfo)
+		{
+			$registerInfoDecoded = json_decode($registerInfo, true);
+			if ($registerInfoDecoded && !$registerInfoDecoded['groupId'])
+			{
+				$registerInfoDecoded['groupId'] = $groupId;
+				$registerInfoEncoded = str_replace('\/', '/', json_encode($registerInfoDecoded));
+				$kuser->setRegistrationInfo($registerInfoEncoded);
+				$kuser->save();
+				print_r('Adding groupId: ' . $groupId . ' to RegistrationInfo on kuserId: '. $kuserId . "\n");
+			}

```

Comment:
Release memory allocations

Reply:
@ravitshalem  we plan to run it on ~18k users so I think this is crucial. Can you please ping me tomorrow once it fixed? we plan to run it tomorrow morning. Thanks!